routers/matchmaking.js

```diff
@@ -48,49 +48,126 @@ router.get("/:room_id/:subroom_id/all-instances", middleware.authenticateDevelop
     }
 });
 
+router.get("/my-instance-info", middleware.authenticateToken, async (req, res) => {
```

Comment:
New endpoint used for determining what kind of instance to create when traversing a Subroom Link.